foodgroup/icbm_test.go

```diff
@@ -536,8 +542,9 @@ func TestICBMService_EvilRequest(t *testing.T) {
 			},
 		},
 		{
-			name:                "don't let users block themselves",
+			name:                "don't let users warn themselves",
 			senderSession:       newTestSession("sender-screen-name"),
+			recipientSession:    newTestSession("sender-screen-name"),
```

Comment:
recipient session can actually be `nil` here because the check for equality is done by comparing the screen names.